.gitignore

```diff
@@ -0,0 +1,2 @@
+secret/.token
+secret/.db_conn
```

Comment:
A little improvement: just `secret` also works

Reply:
I often switch between different laptops and/or its OSs. And .gitignore works for me as a note about missing files :)